specs/test_formats/genesis/validity.md

```diff
@@ -4,12 +4,18 @@ Tests if a genesis state is valid, i.e. if it counts as trigger to launch.
 
 ## Test case format
 
-```yaml
-description: string    -- description of test case, purely for debugging purposes
-bls_setting: int       -- see general test-format spec.
-genesis: BeaconState   -- state to validate.
-is_valid: bool         -- true if the genesis state is deemed valid as to launch with, false otherwise.
-```
+### `genesis.yaml`
+
+A `BeaconState`, the state to validate as genesis candidate.
+
+Also available as `genesis.ssz`. 
+
+### `is_valid.yaml`
+
+A boolean, true if the genesis state is deemed valid as to launch with, false otherwise.
```

Comment:
`.ssz` available?

Reply:
It's a bare boolean. That would be a 1 byte file. I think everyone can parse small yaml fine for small data like this. Will reconsider if teams like SSZ for even this, but expect yaml to be around for a while.